src/StackExchange.Redis/ResultProcessor.cs

```diff
@@ -1325,7 +1325,7 @@ protected override bool SetResultCore(PhysicalConnection connection, Message mes
                 {
                     case ResultType.MultiBulk:
                         var final = result.ToArray(
-                                (in RawResult item, in ChannelState state) => item.AsRedisChannel(state.Prefix, state.Mode),
+                                (in RawResult item, in ChannelState state) => item.AsRedisChannel(state.Prefix, state.Mode, isSharded: false),
```

Comment:
do we need to support `PUBSUB SHARDCHANNELS` (see `SubscriptionChannels[Async]`) ?

Reply:
ditto `SHARDNUMSUB` - see `SubscriptionPatternCount` / `SubscriptionSubscriberCount`

Reply:
yes, plan to add it in a separate PR.

Reply:
ssub handled in my extensions to this PR